etrago/cluster/networkclustering.py

```diff
@@ -178,25 +300,25 @@ def cluster_on_extra_high_voltage(network, busmap, with_time=True):
         Container for all network components of the clustered network.
     """
 
-    network_c = Network()  
+    network_c = Network()
 
-    network, busmap = adjust_no_electric_network(network, busmap)
+    network, busmap = adjust_no_electric_network(network, busmap, cluster_met= 'ehv')
 
     buses = aggregatebuses(
-        network, busmap, {
-            'x': _leading(
-                busmap, network.buses), 'y': _leading(
-                busmap, network.buses)})
+        network,
+        busmap,
+        {"x": _leading(busmap, network.buses), "y": _leading(busmap, network.buses)},
+    )
 
     # keep attached lines
     lines = network.lines.copy()
     mask = lines.bus0.isin(buses.index)
     lines = lines.loc[mask, :]
-    
+
     # keep attached links
     links = network.links.copy()
-    dc_links = links[links['carrier'] == 'AC']
-    links = links[links['carrier'] != 'AC']
+    dc_links = links[links["carrier"] == "AC"]
+    links = links[links["carrier"] != "AC"]
```

Comment:
DC-links have carrier =  'DC'

Reply:
Solved in commit 98adb4a5b1c29ffcac1441e9081caec342a31caf.